draw_graph error when loading model object that has nodes with schematic positions attributes

```
            node_dict["clss"] = [cls.__name__.lower() for cls in classes]
            try:
>               node["position"] = node.position["schematic"]
E               TypeError: 'Reservoir' object does not support item assignment

pywr\notebook\__init__.py:75: TypeError
```
The position is wrongly assigned to the node object rather than `node_dict`. This is not picked up in the tests as the function is tested on a model that does not include any nodes with schematic position attributes.

--- a/pywr/notebook/__init__.py
+++ b/pywr/notebook/__init__.py
@@ -72,7 +72,7 @@ def pywr_model_to_d3_json(model, attributes=False):
         classes = classes[::-1]
         node_dict["clss"] = [cls.__name__.lower() for cls in classes]
         try:
-            node["position"] = node.position["schematic"]
+            node_dict["position"] = node.position["schematic"]
         except KeyError:
             pass
 
@@ -128,7 +128,7 @@ def get_node_attr(node):
         else:
             attr_val = str(attr_val)
 
-        attribute_data.append({"attribute": attr_name,  "value": attr_val})
+        attribute_data.append({"attribute": attr_name, "value": attr_val})
 
     return attribute_data
 

--- a/tests/models/demand_saving2_with_variables.json
+++ b/tests/models/demand_saving2_with_variables.json
@@ -19,7 +19,10 @@
             "type": "reservoir",
             "name": "Reservoir",
             "max_volume": 1000,
-            "initial_volume": 1000
+            "initial_volume": 1000,
+            "position": {
+                "schematic": [1, 1]
+            }
         },
         {
             "type": "output",

--- a/tests/test_notebook.py
+++ b/tests/test_notebook.py
@@ -1,4 +1,5 @@
 import pytest
+from numpy.testing import assert_array_equal
 
 try:
     import IPython
@@ -43,6 +44,9 @@ def test_from_json(from_json):
     node_names = ["Inflow", "Reservoir", "Demand", "Spill"]
     for node in json_dict["nodes"]:
         assert node["name"] in node_names
+
+        if node["name"] == "Reservoir":
+            assert_array_equal(node["position"], [1, 1])
 
     demand = get_node(json_dict["nodes"], "Demand")
     demand_max_flow = get_node_attribute(demand, "max_flow")

